src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
```

Comment:
So to follow up on this: do we need this or not? It seems like the current implementation isn't necessarily wrong. Yes, it does seem weird that orchestration would return success even if the call to rawls eventually failed, but there's nothing that needs to rely on this status code from orch if the call to rawls has loggly alerts setup. Loggly would just be alerting from 1 layer instead of 2.